Trace.py

```diff
@@ -31,14 +31,22 @@ def _addHTML(xx):
     f.write("<td>%s</td>" % x)
   f.write("</tr>\n")
   f.close()
-f = open(debugLog,"w")
-f.write("<html><head><style>body { font-family: sans-serif; background:#333; color: #ccc } img { border: 1px solid #FFF; } td, tr, table { background: #444; padding: 10px; border:1px solid #888; border-collapse: collapse; }</style></head><body><table>\n")
-#FIXME: atexit close tags.. but yolo!
-f.close()
-_addHTML(["<b>#</b>", "<b>Opcode / Method</b>", "..."])
 
+def setupOutput(outputDir):
+  global OutputDir
+  global debugLog
 
-def _htmlPrint(s):
```

Comment:
The `_` indicates that this is a method not intended to be used outside of this module and is intentional. In #38 you'll see that many more methods are underscore prefixed to clarify the public API.

Reply:
My bad, fixed.